pkg/cmd/eventlistener/describe.go

```diff
@@ -0,0 +1,219 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eventlistener
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/cli/pkg/validate"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+	"sigs.k8s.io/yaml"
+)
+
+const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .EventListener.Namespace }}
+
+{{- if ne .EventListener.Spec.ServiceAccountName "" }}
+{{decorate "bold" "Service Account"}}:	{{ .EventListener.Spec.ServiceAccountName }}
+{{- end }}
+
+{{- if ne .EventListener.Spec.ServiceType "" }}
+{{decorate "bold" "Service Type"}}:	{{ .EventListener.Spec.ServiceType }}
+{{- end }}
+
+{{- $value := getURL .EventListener }}
+{{-  if ne $value "" }}
+{{decorate "bold" "URL"}}:	{{ $value }}
+{{- end }}
+
+{{- $value := getEventListenerName .EventListener }}
+{{-  if ne $value "" }}
+{{decorate "bold" "EventListnerServiceName"}}:	{{ $value }}
+{{- end }}
+
+{{- if eq (len .EventListener.Spec.Triggers) 0 }}
+
+ No EventListenerTriggers
+{{- else }}
+{{ " " }}
+{{- range $v := .EventListener.Spec.Triggers }}
+{{decorate "cyan" "EventListenerTriggers\n"}}
```

Comment:
Sorry, I missed this, but we shouldn't use color here. Probably just make it bold.

Reply:
Done